### DISCUSSION TITLE
Cover Art Oversized

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1749] [Bug Reports]
I use a 7" Android tablet with just cover art and top/bottom bars displayed on the playback page. The current update over-sizes some cover art so that it is partly covered by the bottom bar.

**Comment by R101G** [comment]
@rern 
Some cover art is still oversized. The variation when using the radio streams makes the UI very untidy. Would it be possible to revert to the previous sizing, or limit the cover art to the size of the default graphic?

**Comment by R101G** [comment]
@rern
Apart from the look, this is a real nuisance when swiping or picking up the tablet. It is far too easy to skip tracks by accidentally touching the full-width cover art.

**Reply by rern** [reply]
A screenshot please.

**Reply by R101G** [reply]
@rern,

Sorry, I missed this request. The problem is not there in landscape mode, or if I add a time display to the playback page.

![Screenshot](https://github.com/user-attachments/assets/0afc16db-9202-4866-8631-11a695ad6aeb)

**Comment by R101G** [comment]
@rern 
This problem is still present in 20241129. Is there anything I can do myself to restore the previous sizing?

**Comment by R101G** [comment]
@rern 
After 3 months and a lot of updates, I am wondering if this is a "will not fix" situation. Please can you reassure me that it is still on your to do list?

This question is prompted by my skipping to the last track again when picking up the tablet.

Thanks.

**Reply by rern** [reply]
Try the UPDATE.

**Reply by R101G** [reply]
@rern 
Thanks for looking at this, but the UPDATE has made no noticeable difference.

**Reply by rern** [reply]
![01](https://github.com/user-attachments/assets/cc8603cc-ae6a-4a72-8f29-19ca7817fda3)

**Reply by R101G** [reply]
@rern 
There is no problem if a volume control or time display is added to the playback page. I only have the album art.

![Screenshot_20241208-124206](https://github.com/user-attachments/assets/f00a0fdb-5328-4e86-8e8e-df0311584ad7)

**Reply by R101G** [reply]
This is how it looks with time added (and how it used to be, I think)
Edit: Note how it has accidentally skipped to the last track!

![Screenshot_20241208-125030](https://github.com/user-attachments/assets/c6ef4fc1-0e5e-4065-8b97-0497762e8af0)

**Reply by rern** [reply]
Please try the UPDATE.

**Comment by R101G** [comment]
@rern 
Thanks rern, but it is still the same. Some cover art is OK, but some is right up to the edge of the screen. It used to be OK - the size of cover art changed, but was never this large. As nobody else is complaining, it must be because I have cover art only, on a 7" 1024x600 screen. As you see, if I add time or volume display to the playback page, the size is fine.

**Comment by R101G** [comment]
@rern 
The bottom bar is now transparent, so touching the playback icon stops the player when oversized coverart is displayed.

**Comment by R101G** [comment]
Seems to have been fixed? - Thanks.

**Comment by R101G** [comment]
@rern 
Sorry rern, my mistake. I still get some coverart overlapping the bottom bar and touching the sides in portrait mode.

**Comment by R101G** [comment]
@rern This problem does not seem to have the same source as  #1885 does, so after trying the current UPDATE, I am posting here. 

My problem is that the cover art size varies wildly in portrait mode if it is the only item on the playback page. I am asking for whatever code controls the image size when the time or volume control are displayed on the playback page, to control the image size in exactly the same way as it does when time or volume are displayed.

The image size is controlled in landscape mode - this bug only affects portrait mode.

**Comment by R101G** [comment]
Problem is still present in 20250516, so will close this and open a feature request.
